src/components/DefineCompany/AddNameTranslation.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div id="name-translation">
+    <!-- Name Translation form -->
+    <v-form v-model="nameTranslationForm" ref="ntForm" name="name-translation-form">
```

Comment:
You don't need the ref unless you're going to reference the form (eg, for validation or reset).

Reply:
ahh thanks, will remove!

Reply:
Yeah. You only need a ref to reference the form in script, like this:
`this.$refs.addPersonOrgForm.reset()`
or this
`this.$refs.dateTimeForm.validate()`

Reply:
Oh nice, yea that seems really helpful when required!